AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockAssertUserLoggedActionGroup.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAdobeStockAssertUserLoggedActionGroup">
+        <waitForPageLoad stepKey="waitForLoad"/>
```

Comment:
I don't think `waitForPageLoad` is necessary for these action groups. For example, the proper `waitForPageLoad` use in my opinion is placing it as a next line after navigation. So if this action group fails because of page is not loaded  - `waitForPageLoad` should be moved to an action group responsible for opening the page.

Reply:
ok